src/main/java/com/amihaiemil/charles/github/Language.java

```diff
@@ -26,21 +26,23 @@
 
 package com.amihaiemil.charles.github;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import java.io.IOException;
 import java.util.Properties;
 import java.util.Set;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 /**
  * Language that the agent speaks.
  * @author Mihai Andronache (amihaiemil@gmail.com)
  * @version $Id$
  * @since 1.0.0
- * 
+ *
+ * @todo #174:30m/DEV Add commands_ru.properties files for commands and responses
```

Comment:
@salikjan 
> What is the meaning of index command in file command_en? Is it a noun or a verb? In the meaning of list or a pointer?

It is a verb: "please, index this page". When it comes to search-engines, and science of search in general, first step is indexing of content... it's an abstract concept which can be implemented different ways. One example of "indexing" a page of text would be: 

You take all the "salikjan" occurences from the page and note them down something like

```
salikjan
- 3 appearances
- 1 on line 7
- 1 on line 10
- 1 on line 15
```
something like making the Index table (chapter's list) for a book.

In our case, when the bot is instructed to "index", it simply crawls the website and sends the content to ElasticSearch for actual indexing.